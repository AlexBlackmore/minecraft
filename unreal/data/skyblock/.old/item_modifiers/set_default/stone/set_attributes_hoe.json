{
	"function":"set_attributes",
	"modifiers":[
		{
			"name":"Attack Speed",
			"attribute":"generic.attack_speed",
			"operation":"addition",
			"amount":-2,
			"slot":"mainhand"
		}
	]
}
