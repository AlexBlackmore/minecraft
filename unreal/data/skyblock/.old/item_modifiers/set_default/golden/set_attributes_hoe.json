{
	"function":"set_attributes",
	"modifiers":[
		{
			"name":"Attack Speed",
			"attribute":"generic.attack_speed",
			"operation":"addition",
			"amount":-3,
			"slot":"mainhand"
		}
	]
}
